freesound/static/webpack/common.js

```diff
@@ -0,0 +1,56 @@
+const path = require('path');
+
+const rootPath = path.resolve('.');
+
+const SUPPORTED_BROWSERS = [
```

Comment:
This is probably the most important line in the entire PR @ffont @alastair

Reply:
It sets up both css autoprefixing and js transpiling according to this setting

Reply:
I think we said we can drop IE < 10 (or even < 11) so I think this list is fine. In any case if this thing lets us know when things won't be supported in non-supported browsers (like with a warning or something), then we'll be better able to decide whether we want to have a fallback implementation of something or just leave it totally unsupported. For example, maybe we have a super basic version of the player for old browsers so these can still be used for basic navigation and for playing sounds.